2020-02-Spring/0_arrays_matrix/README.md

```diff
@@ -26,21 +26,56 @@ In the style of:
 
 <a name="p1"/>
 
-### 1. PROBLEM 1 TODO :bug:
+### 1. Best Time to Buy and Sell Stock
 
-Source: TODO :bug:
+Source: [LeetCode](https://leetcode.com/problems/best-time-to-buy-and-sell-stock/)
 
 #### Scenario
 
-Problem Statement TODO :bug:
+Say you have an array of n elements representing stock prices.
```

Comment:
Seems like and interesting problem. I'll submit a full review after you complete the question. Be sure to write down in the problem statement what format you want the output in, and whether there are any guarantees on the input set (i.e. no negatives, no nulls, w/e)

Reply:
I will finish it up first thing tomorrow. Thanks.